Src/AutoFixture.NUnit3/AutoDataAttribute.cs

```diff
@@ -48,6 +49,7 @@ public IEnumerable<TestMethod> BuildFrom(IMethodInfo method, Test suite)
             yield return test;
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
```

Comment:
According to [its documentation](https://msdn.microsoft.com/en-us/library/ms182137), this particular rule mustn't be suppressed, but I agree that it still should, in this particular case. I would like, however, a `Justification` text.